routes/users.js

```diff
@@ -0,0 +1,68 @@
+const express = require('express');
+const { keysToCamel } = require('../common/utils');
+const { db } = require('../server/db');
+
+const userRouter = express.Router();
+
+userRouter.get('/', async (req, res) => {
+  try {
+    const allUsers = await db.query(`SELECT * FROM users;`);
+    res.status(200).json(keysToCamel(allUsers));
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
+
+userRouter.get('/pending-accounts', async (req, res) => {
+  try {
+    const pendingAccounts = await db.query(`SELECT * FROM users WHERE approved = FALSE;`);
+    res.status(200).json(keysToCamel(pendingAccounts));
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
+
+userRouter.post('/', async (req, res) => {
+  try {
+    const { id, email, type, approved } = req.body;
+    await db.query(`INSERT INTO users (id, email, "type", approved) VALUES ($1, $2, $3, $4);`, [
+      id,
+      email,
+      type,
+      approved,
+    ]);
+    res.status(201).json({
+      status: 'Success',
```

Comment:
instead of returning success, can you return the id of the object that was just created?